source/publishing_data/08_configuring_dataset_export/configuring_dataset_export.rst

```diff
@@ -26,3 +26,32 @@ By default, the records of a dataset export are not ordered. It is possible to c
    :class: important
 
    The sort does not apply on additional export formats.
+
+Adding other geographical projections
+-------------------------------------
+
+The geographical coordinates of a published dataset are by default expressed in WGS84, no matter what was their original projection when creating the dataset. It is possible to configure additional geographical projections to enable users to export the dataset in their preferred geographical projection.
+
+1. In the Additional geographical projections area, click on the Additional projections textbox.
+2. Write the EPSG code of the chosen geographical projection. Or, choose one from the suggestions list.
+
+Adding more export formats
+--------------------------
+
+More specific formats can be available for export:
+
+- GPX (default): this format can directly be configured by the administrators of the domain to be available as additional export format.
```

Comment:
The "by the administrators" is superfluous, it can be configured by anyone with edit permissions on the dataset.
You can clarify that the GPX extractor requires a geographic field.